lib/translate/BUILD

```diff
@@ -0,0 +1,11 @@
+cc_library(
+  name = "status_translator_lib",
```

Comment:
Not sure whether the best option is to have separate libraries for each translator or to just have one translator library.

Reply:
I vote for having them all in one library called "translator_lib"

Reply:
I added a more extensive comment in a different thread, but I think the cleaner approach is to have each translator in its own cc_library target, and then have one big cc_library target that includes all of the individual targets.